spec-main/api-remote-spec.ts

```diff
@@ -641,6 +641,16 @@ ifdescribe(features.isRemoteModuleEnabled())('remote module', () => {
       const { functionWithToStringProperty } = require('electron').remote.require(path.join(fixtures, 'to-string-non-function.js'));
       expect(functionWithToStringProperty.toString).to.equal('hello');
     });
+
+    const protocolKeys = Object.getOwnPropertyNames(protocol);
+    remotely.it(protocolKeys)('remote.protocol returns all keys', (protocolKeys: [string]) => {
+      const protocol = require('electron').remote.protocol;
+      const remoteKeys = Object.getOwnPropertyNames(protocol);
+      expect(remoteKeys).to.deep.equal(protocolKeys);
+      for (const key of remoteKeys) {
+        expect(typeof (protocol as any)[key]).to.equal('function');
+      }
+    });
```

Comment:
Please add this to https://github.com/electron/remote/blob/master/test/all.ts as well 🙇